crates/bevy_render/Cargo.toml

```diff
@@ -28,34 +28,44 @@ webgl = ["wgpu/webgl"]
 
 [dependencies]
 # bevy
-bevy_app = { path = "../bevy_app", version = "0.8.0-dev" }
-bevy_asset = { path = "../bevy_asset", version = "0.8.0-dev" }
-bevy_core = { path = "../bevy_core", version = "0.8.0-dev" }
-bevy_derive = { path = "../bevy_derive", version = "0.8.0-dev" }
-bevy_ecs = { path = "../bevy_ecs", version = "0.8.0-dev" }
-bevy_encase_derive = { path = "../bevy_encase_derive", version = "0.8.0-dev" }
-bevy_math = { path = "../bevy_math", version = "0.8.0-dev" }
-bevy_mikktspace = { path = "../bevy_mikktspace", version = "0.8.0-dev" }
-bevy_reflect = { path = "../bevy_reflect", version = "0.8.0-dev", features = ["bevy"] }
-bevy_render_macros = { path = "macros", version = "0.8.0-dev" }
-bevy_transform = { path = "../bevy_transform", version = "0.8.0-dev" }
-bevy_window = { path = "../bevy_window", version = "0.8.0-dev" }
-bevy_utils = { path = "../bevy_utils", version = "0.8.0-dev" }
+bevy_app = { path = "../bevy_app", version = "0.7.0" }
```

Comment:
This shouldn't be in here; looks like a rebasing issue.

Reply:
I used the same old rebase workflow I always use, and it absolutely exploded the commit history. :anger:

Reply:
Should be all fixed now though. A diff shows only changes I expect to see.